packages/react-components/src/types/VideoGalleryParticipant.ts

```diff
@@ -108,6 +108,11 @@ export interface VideoGalleryLocalParticipant extends VideoGalleryParticipant {
    * @beta
    * */
   reaction?: Reaction;
+  /* @conditional-compile-remove(spotlight) */
+  /**
+   * Whether local participant has reacted
+   * */
+  capabilities?: { spotlightParticipant?: { isPresent: boolean } };
```

Comment:
I'm worried this naming is too tied to the calling sdk. As a developer just using the video gallery, its not clear that this controls whether or not there is a button for the local participant to spotlight themself. This participant could also be spotlighted by someone else despite having this set to false.

Ideating other options:
- `VideoGallery.showLocalSpotlightButton`
- `VideoGallery.showLocalSpotlightControls`
- `VideoGallery.VideoGalleryLocalParticipant.disableSelfSpotlightControls`
- Split `start/StopSpotlight` into `start/stopRemoteParticipantSpotlight` and `start/stopLocalParticipantSpotlight` and hence don't supply the `startLocalParticipantSpotlight` if the capbility is not there at the SDK level

Reply:
I like the last point because it's very clear what the callbacks are for and it is intuitive that if they are not provided the menu item will not be available on remote/local tile. I did think of something along the same lines but 4 callback props seemed like a lot to add to VideoGallery. Also I can use the capability to either provide these callbacks or not and no notion of capability will be used in the VideoGallery component. Thanks!

Side note: I originally wanted to use the absence of the capability to hide start/stop spotlight menu for remote participants and start for the local participant. The local participant should always be able to exit spotlight. But you're right that the consequences of this capability is not clear